### ISSUE TITLE
Type inside a bounding box

### ISSUE THREAD

**Issue by adamjarling** [adamjarling/loc-speculative-annotations#66]
Users to type inside of a bounding text box that automatically wraps text when they adjust the width of the box.

Need to determine if Fabric JS supports this behavior.

**Comment by CourtneyMcClellan** [comment]
@adamjarling  any updates on this? I was working on an annotation today. Having the ability to wrap text will make longer notes much easier to carry out. I would like to move this to MVP but I want to hear from you about what this would entail.

**Comment by jaimemears** [comment]
deleting "nice to have" label

**Comment by adamjarling** [comment]
@CourtneyMcClellan @jaimemears Playing around with the Textbox bounding box, and this is about as good as I can get it on an initial pass.  A few things to note:

- I cannot seem to add the bounding box controls when the textbox is in "editing" mode.
- I reduced the background opacity from `0.5` to `0.2`.  I've noticed other comparable apps off some kind of visual cue of the actual box they're typing in.
- Resizing the box horizontally keeps text same size, and pushes text down onto new lines or extends to a single line.
- Resizing the box vertically or diagonally proportionately increases text size.

I'd say play around with it and let me know whether this is better or worse than before?

![image](https://user-images.githubusercontent.com/3020266/117587393-61372200-b0e3-11eb-809b-a0e8f7bff82c.png)

![image](https://user-images.githubusercontent.com/3020266/117587416-79a73c80-b0e3-11eb-9f32-f43a413fe779.png)

![image](https://user-images.githubusercontent.com/3020266/117587428-89bf1c00-b0e3-11eb-8893-ef1ee2c94304.png)

![image](https://user-images.githubusercontent.com/3020266/117587451-a65b5400-b0e3-11eb-847f-4ead1ec4c0c2.png)
